src/basic/calendarspec.c

```diff
@@ -147,18 +150,21 @@ int calendar_spec_normalize(CalendarSpec *c) {
         return 0;
 }
 
-_pure_ static bool chain_valid(CalendarComponent *c, int from, int to) {
+_pure_ static bool chain_valid(CalendarComponent *c, int from, int to, bool eom) {
```

Comment:
can you add a comment here somewhere that eom means "end of month", so that this is easier to grok for the next person looking at this?